### ISSUE TITLE
--incompatible_strict_action_env

### ISSUE THREAD

**Issue by buchgr** [bazelbuild/bazel#6648]
Starting with Bazel 0.21 the --experimental_strict_action_env option has been renamed to --incompatible_strict_action_env and is now on by default. This means Bazel will no longer use the client's PATH and LD_LIBRARY_PATH environmental variables in the default action environment. If the old behavior is desired, pass --action_env=PATH and --action_env=LD_LIBRARY_PATH. --noincompatible_strict_action_env will also temporarily restore the old behavior. However, as --action_env is a more general and explicit way to pass client environmental variables into actions, --noincompatible_strict_action_env will eventually be deprecated and removed. See #6648 for more details.

Both flag names `--experimental_strict_action_env` and `--incompatible_strict_action_env` will keep working for a few releases.

Also see #2574

cc: @benjaminp

**Comment by benjaminp** [comment]
Do you need to me to update the PR somehow?

**Comment by buchgr** [comment]
No I did the update already. Just had to submit a different change before submitting your change. should be in in a few hours.

**Comment by mattgodbolt** [comment]
I checked bazel v1.1.0 and it seems this flag is still off by default:

```
~/d/research (master|✔) 28.1s $ bazel version
Bazelisk version: v1.1.0
Build label: 1.1.0
Build target: bazel-out/k8-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Mon Oct 21 08:44:00 2019 (1571647440)
Build timestamp: 1571647440
Build timestamp as int: 1571647440
~/d/research (master|✔) 1.1s $ bazel help build | grep incompatible_strict_action
  --[no]incompatible_strict_action_env (a boolean; default: "false")
```

Is this as intended?

**Comment by benjaminp** [comment]
#7026

**Comment by mattgodbolt** [comment]
Thanks @benjaminp :)

**Comment by BoleynSu** [comment]
Hi @buchgr , the default is currently `false` instead of `true`, would you mind updating your comments accordingly?

**Comment by iancha1992** [comment]
A fix for this issue has been included in [Bazel 9.0.0 RC3](https://github.com/bazelbuild/bazel/releases/tag/9.0.0rc3). Please test out the release candidate and report any issues as soon as possible.
If you're using Bazelisk, you can point to the latest RC by setting USE_BAZEL_VERSION=9.0.0rc3. Thanks!
